clevertap-core/src/main/java/com/clevertap/android/sdk/AnalyticsManager.java

```diff
@@ -801,6 +801,25 @@ synchronized void pushDeepLink(Uri uri, boolean install) {
         }
     }
 
+    Future<?> raiseEventForDirectCall(String eventName, JSONObject geofenceProperties) {
```

Comment:
@shivamsharma2710 Rename the method parameter `geofenceProperties` to `dcEventProperties`

Reply:
@darshanclevertap working on it.

Reply:
@darshanclevertap pushed the suggested change.